spec/controllers/sign_up/passwords_controller_spec.rb

```diff
@@ -29,11 +26,8 @@
     end
 
     it 'tracks an invalid password event' do
-      user = create(:user, :unconfirmed)
-      token, = Devise.token_generator.generate(User, :confirmation_token)
-      user.update(
-        confirmation_token: token, confirmation_sent_at: Time.current
-      )
+      token = Devise.token_generator.generate(User, :confirmation_token)
```

Comment:
I don't think that's necessarily bad. This Devise generator returns an array whose first element is the raw token, and the second element is the encrypted token to be stored in the DB. The generator is meant to be used such that each part of the array is assigned to a different variable.

In this case, if you remove the comma, the `token` variable will be an Array, but it will get saved to the DB as a String. Then, when you make the POST request, Devise will compare the passed in token to the value in the DB. In this case, they match, so all is good.

Looking at Devise's `confirmable` module, I see that the confirmation_token doesn't get encrypted. It gets generated via `Devise.friendly_token`, so perhaps we should do the same in this test.